File: Hcs-Platform-App/src/app/models/hcs-basic-model/PlatformUserGroup.ts
import { ApiEntry, Property } from '../../hcs-lib/ApiEntry';

@ApiEntry('api/entity/Hcs.Platform.BaseModels.PlatformUserGroup')
export class PlatformUserGroup {
    @Property()
    PlatformUserId: number;
    @Property()
    PlatformGroupId: number;
}

File: Hcs.Platform.BaseModels/PlatformUserGroup.cs
using System;
using Hcs.Platform.Data;
using Hcs.Platform.User;

namespace Hcs.Platform.BaseModels
{
    public class PlatformUserGroup
    {
        public long PlatformUserId { get; set; }
        public long PlatformGroupId { get; set; }
        public PlatformUser PlatformUser { get; set; }
        public PlatformGroup PlatformGroup { get; set; }

    }
}